bigtable-hbase-1.x-parent/bigtable-hbase-1.x/src/test/java/com/google/cloud/bigtable/hbase1_x/TestMetrics.java

```diff
@@ -56,11 +57,9 @@
   private static final String TEST_PROJECT_ID = "fake-project-id";
   private static final String TEST_INSTANCE_ID = "fake-instance-id";
   private static final TableName TABLE_NAME = TableName.valueOf("fake-table");
-  private static final String FULL_TABLE_NAME =
-      NameUtil.formatTableName(TEST_PROJECT_ID, TEST_INSTANCE_ID, TABLE_NAME.getNameAsString());
   private static Server server;
   private static int dataPort;
-  private static final AtomicInteger callCount = new AtomicInteger(1);
+  private static AtomicInteger callCount = new AtomicInteger(1);
```

Comment:
why not final?

Reply:
Also, I would remove the static state altogether...instantiate a new server for each test...its not that expensive and will avoid flakes in the future

Reply:
thanks - this was accidental when I was testing things out

Reply:
For the static state comment - this would require me reworking the current `@BeforeClass` and `@AfterClass` structure. Do you think that's worth it?

Reply:
I think it should be a fairly easy change...just cut/paste the `@BeforeClass` into '@Before`